_docs/tasks/istio-auth.md

```diff
@@ -9,166 +9,157 @@ type: markdown
-Follow [the Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
+## Before you begin
 
-### Option 2: (recommended) using per-cluster CA
+This task assumes you have:
 
-Only a single Istio CA is deployed for the Kubernetes cluster, in a dedicated namespace.
-Doing this offers the following benefits:
+* Uninstalled Istio for the namespaces you plan to enable Istio Auth.
+To uninstall Istio, please follow the [uninstall instructions]({{home}}/docs/tasks/installing-istio.html#uninstalling).
 
-* In the near future, the dedicated namespace will use
-[Kubernetes RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) (beta in Kubernetes V1.6) to provide security
-boundary. This will offer strong security for Istio CA.
+* Disabled Istio Auth for all namespaces in the Kubernetes cluster.
+Otherwise, this process will deploy a CA that conflicts with existing CAs.
 
-* Services in the same Kubernetes cluster but different namespaces are able to talk to each other through Istio Auth
-without extra trust setup.
+* Completed steps 1 - 4 in [the Istio installation guide]({{home}}/docs/tasks/installing-istio.html#installing-on-an-existing-cluster).
 
-#### Deplying CA
+## Installing Istio Auth
 
-The following command creates  namespace *istio-system* and deploys CA into the namespace:
+### Deploying CA
+
+The Istio CA only needs to be deployed once for the cluster.
+The following command creates a namespace *istio-system* and deploys the cluster-level CA into the namespace:
 
 ```bash
-kubectl apply -f ./kubernetes/istio-auth/istio-cluster-ca.yaml
+kubectl apply -f install/kubernetes/templates/istio-auth/istio-cluster-ca.yaml
 ```
 
-#### <a name="istioconfig"></a>Enabling Istio Auth in Istio config
+### Deploying other services
 
-The following command uncomments the line *authPolicy: MUTUAL_TLS* in the file *kubernetes/istio-X.yaml*,
-and backs up the original file as *istio-X.yaml.bak*
-(*X* corresponds to the Kubernetes server version, choose "15" or "16").
+The following command will enable
+[mTLS](https://en.wikipedia.org/wiki/Mutual_authentication)
+for the services in the "default" namespace,
+and the services are able to use the cluster-level CA deployed in the last step.
+Use the parameter *-n yournamespace* to specify a namespace other than the default one.
 
 ```bash
-sed "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" ./kubernetes/istio-X.yaml > ./kubernetes/istio-auth-X.yaml
+kubectl apply -f install/kubernetes/templates/istio-auth/isio-auth-with-cluster-ca.yaml
 ```
 
-#### Deploying other services
-
-Follow [the general Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
-
-## Disabling Istio Auth
+Follow [the general Istio installation guide]({{home}}/docs/tasks/installing-istio.html) from step 5.
```

Comment:
I think it's step 6 now.

It may be better to say somewhere above, that this page replaces step 5 of the guide, and here Follow the rest of the steps, without giving a number.